src/domains/BridgedDomain.sol

```diff
@@ -15,7 +15,7 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 pragma solidity >=0.8.0;
 
-import { Domain } from "./Domain.sol";
+import { Domain, StdChains } from "./Domain.sol";
```

Comment:
Any specific reason to use this contract to pass `StdChains` and not importing directly in the destination from forge-std?

Reply:
No, I'll change to import directly.

Reply:
done

Reply:
Great!